BRAINSABC/brainseg/BRAINSABC.cxx

```diff
@@ -651,8 +651,11 @@ main(int argc, char ** argv)
                 }
               }
               std::string intraSubjectTransformFileName =
-                outputDir + GetStrippedImageFileNameExtension((*imIt).c_str()) + "_to_" +
-                GetStrippedImageFileNameExtension(GetMapVectorFirstElement(inputVolumeMap)) + suffixstr + ".h5";
+                outputDir.append(GetStrippedImageFileNameExtension((*imIt).c_str()))
```

Comment:
I don't think this is correct. 

Previously outputDir was not changed.  Now the value of outputDIr is changed, then the changed values is copied to the intraSubjectTransformFileName. variable.

Reply:
The value of outputDir is now appended onto an empty string, so its contents should no longer be changed.